plugins/node/opentelemetry-instrumentation-pg/package.json

```diff
@@ -73,8 +73,9 @@
   },
   "dependencies": {
     "@opentelemetry/instrumentation": "^0.53.0",
-    "@opentelemetry/semantic-conventions": "^1.27.0",
+    "@opentelemetry/semantic-conventions": "1.27.0",
```

Comment:
drive-by-fix: we're using the `@opentelemetry/semantic-conventions/incubating` entrypoint which may break in minor versions - pinning this to avoid any surprises when we release the `1.28.0`.

Reply:
Might want the same at https://github.com/open-telemetry/opentelemetry-js-contrib/blob/645ac2e9fa322503e88283985f3f3bb2349a9729/plugins/node/opentelemetry-instrumentation-aws-lambda/package.json#L63

We should have a lint check for this perhaps.